src/modules/helpchan.ts

```diff
@@ -340,6 +340,8 @@ export class HelpChanModule extends Module {
 		await this.ensureAskChannels(msg.guild!);
 
 		this.busyChannels.delete(claimedChannel.id);
+
+		return msg.channel.send(`👌 successfully claimed ${claimedChannel}.`);
```

Comment:
Thanks! Two small things:

1. It should be `claimedChannel.name`
2. Remove the fullstop for consistency's sake

Reply:
Ok, the linked issue suggested I should use the channel mention format, I can change it. I added a fullstop because I saw other commands had one too (i.e. `resolved`), but doesn't seem to be consistent across the board. I can remove it.

Reply:
Ahh I see! So sorry, forgot about this. If you can revert the `.name` change I'll merge this 😅

Reply:
Done!